### ISSUE TITLE
Unable to log in

### ISSUE THREAD

**Issue by Che-debarras** [Sandmann79/xbmc#508]
**Addon used**
- [x] Amazon VOD (plugin.video.amazon-test)
- [ ] Browser Launcher (plugin.program.browser.launcher)

- Addon version: latest

**Account type**
- [x] primevideo.com
- [ ] amazon.(com/co.uk/de/jp)

**System Setup (please provide the following information):**
- Hardware: Box tv Vorke Z6+ , Kingbox K3
- OS version: Android 7.1
- Kodi version number: Leia 18.8

**Upload Logs**

Unable to connect to the account from android device, no connection's windows but i tried the discountinued version and the window of connection appears.

**Comment by Sandmann79** [comment]
There must be something wrong with your installation. The most recent version of urllib3 is 1.25.6 for leia. You have 1.25.8 installed. 

Try to install the official one from here:
https://mirrors.kodi.tv/addons/leia/script.module.urllib3/

**Comment by Che-debarras** [comment]
> There must be something wrong with your installation. The most recent version of urllib3 is 1.25.6 for leia. You have 1.25.8 installed.
> 
> Try to install the official one from here:
> https://mirrors.kodi.tv/addons/leia/script.module.urllib3/

I installed the offcial one but same issue unable to have the connection Window

**Comment by Sandmann79** [comment]
It's still the same error in the log file?

**Comment by Che-debarras** [comment]
> It's still the same error in the log file?

Yes the same error, when I click to log in it is said amazon vod error and if I check the log file it is the same error in the log file

**Comment by Che-debarras** [comment]
I downloaded the latest modification you did 5hours ago but the first issue appeared again : no connection windows with this 0.8.9 Beta

**Comment by Varstahl** [comment]
The last login related patches were more than 2 weeks ago, recent ones are localization and new features for primevideo, not related to login issues.

Make sure you have "Settings > Connection > Disable SSL certificate validation" disabled (not checked), try "Remove credentials and cookies" and then "Sign in…". If it's an SSL related issue, it should go away.

**Comment by Che-debarras** [comment]
@Varstahl 

Your solution is worse , now i dont have anymore the window connection even if I replace the files from the previous update.

Please,  if you dont know how to fix it just ignore my issue because it is worse with your bad advices.

**Comment by Varstahl** [comment]
hahaha, sure, I'm only the main developer of PrimeVideo, it's clear you know better, I'll leave you to it.

**Comment by Che-debarras** [comment]
> hahaha, sure, I'm only the main developer of PrimeVideo, it's clear you know better, I'll leave you to it.

I never said I know better than you but from what I read on your message you dont know better than me to make the issue worse than before.

Please yes leave us with our issue if you dont have the solution.

thanks


To go back to the issue for someone with knowledge to help:
 
The window of connnection doesnt appear
From the log file in my previous message I said I manage to have it
But it was after updates made this morning or yesterday on github
And I l noticed that the files updated make my addon a 0.8.3 beta version 
It asks me to update but if I update to 0.8.8 or 0.8.9 beta the window connection doesnt appear.

With 0.8.3 bêta I log in but notjing happen and I I have a log error saying 
No name module Xlib display
Disable ssl by checked it make the issue worse and I I have to delete the add and install it again

**Comment by Sandmann79** [comment]
v0.8.3~beta didn't contain/need the Poolmanager module. 
Where you get this version - i don't know. But it can never be from yesterday.

It's better to test the Addon from a clean installation. The easiest way to do this, is to rename `/storage/emulated/0/Android/data/org.xbmc.kodi` to something like this `/storage/emulated/0/Android/data/org.xbmc.kodi-bak`, start Kodi and install only the repo for this addon - nothing more.

It's like @Varstahl wrote, the last commit regarding to the login process is this here: 88dbc10a4c86eaa37933989ac7749a5fc19905a0.

**Comment by ch3573r** [comment]
Damn, he's rude...

But, I just installed Kodi on Linux Mint via ppa and your addon via the Repo, I also do not have Login Window on the Amazon App.

**Comment by Che-debarras** [comment]
> Damn, he's rude...
> 
> But, I just installed Kodi on Linux Mint via ppa and your addon via the Repo, I also do not have Login Window on the Amazon App.

I am not rude but the developers waste our time and doesn't read the log.
@ch3573r you will see there don't want to fix it or try to see where is the issue

I updated with the files from github and it makes the version 0.8.3 beta
And the main dev doesn't know the version of files is uploaded on their github
I gave the logs and despite I describe the log they are unable to see the bug

It is my last message because I know it is useless to report issues and dont waste your time @ch3573r

**Comment by Sandmann79** [comment]
@ch3573r this could be fixed with 88dbc10a4c86eaa37933989ac7749a5fc19905a0. If not, please open a new issue with logs.
